lib/sanbase_web/graphql/resolvers/clickhouse_resolver.ex

```
@@ -36,6 +39,34 @@ defmodule SanbaseWeb.Graphql.Resolvers.ClickhouseResolver do
     end
   end
 
+  def daily_active_deposits(_root, args, _resolution) do
```

In most places we pattern match the args in the function head. It is arguably nicer to read and you'll fail early with a more meaningful error when not correct params are given.